app/assets/javascripts/admin/addon/routes/admin.js

```diff
@@ -1,21 +1,34 @@
 import { tracked } from "@glimmer/tracking";
 import { service } from "@ember/service";
+import KeyboardShortcuts, {
+  PLATFORM_KEY_MODIFIER,
+} from "discourse/lib/keyboard-shortcuts";
 import DiscourseRoute from "discourse/routes/discourse";
 import { i18n } from "discourse-i18n";
+import AdminSearchModal from "admin/components/modal/admin-search";
 
 export default class AdminRoute extends DiscourseRoute {
   @service sidebarState;
   @service siteSettings;
   @service store;
   @service currentUser;
   @service adminSidebarStateManager;
+  @service modal;
   @tracked initialSidebarState;
 
   titleToken() {
     return i18n("admin_title");
   }
 
   activate() {
+    KeyboardShortcuts.addShortcut(
+      `${PLATFORM_KEY_MODIFIER}+/`,
+      () => this.showAdminSearchModal(),
+      {
+        global: true,
+      }
+    );
```

Comment:
This is never unregistered, so if I go to the admin dashboard and then to the forum part, this shortcut will still be active.

Reply:
Ah yes good point, will fix